scripts/tools/generate_esp32_chip_factory_bin.py

```diff
@@ -268,18 +264,24 @@ def validate_args(args):
 def gen_spake2p_params(passcode):
     iter_count_max = 10000
     salt_len_max = 32
+    salt = base64.b64encode(os.urandom(salt_len_max)).decode('utf-8')
+
+    spake2p_py = '{}/{}'.format(CHIP_TOPDIR, 'scripts/tools/spake2p/spake2p.py')
 
     cmd = [
-        TOOLS['spake2p'], 'gen-verifier',
+        spake2p_py, 'gen-verifier',
```

Comment:
Could it have been used as a direct import instead?

Reply:
Sounds like better approach, importing spake2p now!